sandbox/src/game.ts

```diff
@@ -236,49 +384,69 @@ follower.actions
+};
+
+var healthbar2 = new ex.Graphics.Rectangle({
+  width: 140,
+  height: 5,
+  color: new ex.Color(0, 255, 0)
+});
 
-// Add Title above player
-var playerLabel = new ex.Label({
-  text: 'My Player',
-  pos: new ex.Vector(-70, -69),
-  fontFamily: 'Times New Roman'
-  // spriteFont: spriteFont
+var backroundLayer = player.graphics.layers.create({
+  name: 'background',
+  order: -1
 });
 
-player.add(playerLabel);
+backroundLayer.show(healthbar2, { offset: ex.vec(0, -70) });
+var playerText = new ex.Graphics.Text({
+  text: 'A long piece of text is long',
+  font: new ex.Graphics.Font({
+    size: 20,
+    family: 'Times New Roman'
+  })
+});
+// playerText.showDebug = true;
+backroundLayer.show(playerText, { offset: ex.vec(0, -70) });
 
 // Retrieve animations for player from sprite sheet
-var left_sprites = spriteSheetRun.getAnimationBetween(game, 1, 11, 50).sprites;
-var left = new ex.Animation({
-  engine: game,
-  speed: 50,
-  sprites: left_sprites
-});
+var left = ex.Graphics.Animation.fromSpriteSheet(spriteSheetRun, ex.Util.range(1, 10), 50);
 // var left = new ex.Animation(game, left_sprites, 50);
-var right = spriteSheetRun.getAnimationBetween(game, 1, 11, 50);
+var right = left.clone(); // spriteSheetRun.getAnimationBetween(game, 1, 11, 50);
 right.flipHorizontal = true;
-var idle = spriteSheetRun.getAnimationByIndices(game, [0], 200);
+var idle = ex.Graphics.Animation.fromSpriteSheet(spriteSheetRun, [0], 200); // spriteSheetRun.getAnimationByIndices(game, [0], 200);
 //idle.anchor.setTo(.5, .5);
-var jumpLeft = spriteSheetJump.getAnimationBetween(game, 0, 11, 100);
-var jumpRight = spriteSheetJump.getAnimationBetween(game, 11, 22, 100);
-left.loop = true;
-right.loop = true;
-idle.loop = true;
-
-jumpRight.freezeFrame = 0;
-jumpLeft.freezeFrame = 11;
+var jumpLeft = ex.Graphics.Animation.fromSpriteSheet(
+  spriteSheetJump,
+  ex.Util.range(0, 10).reverse(),
+  100,
+  ex.Graphics.AnimationStrategy.Freeze
+); // spriteSheetJump.getAnimationBetween(game, 0, 11, 100);
+var jumpRight = ex.Graphics.Animation.fromSpriteSheet(spriteSheetJump, ex.Util.range(11, 21), 100, ex.Graphics.AnimationStrategy.Freeze); // spriteSheetJump.getAnimationBetween(game, 11, 22, 100);
+// left.loop = true;
+// right.loop = true;
+// idle.loop = true;
+
+// jumpRight.freezeFrame = 0;
+// jumpLeft.freezeFrame = 11;
 
 // Add animations to player
-player.addDrawing(Animations.Left, left);
-player.addDrawing(Animations.Right, right);
-player.addDrawing(Animations.Idle, idle);
-player.addDrawing(Animations.JumpRight, jumpRight);
-player.addDrawing(Animations.JumpLeft, jumpLeft);
+player.graphics.add(Animations.Left, left);
+player.graphics.add(Animations.Right, right);
+player.graphics.add(Animations.Idle, idle);
+player.graphics.add(Animations.JumpRight, jumpRight);
+player.graphics.add(Animations.JumpLeft, jumpLeft);
 
 // Set default animation
-player.setDrawing(Animations.Idle);
+player.graphics.swap(Animations.Idle);
```

Comment:
Wonder if `.switchTo` or `.use` could be a nicer API?

Reply:
I think I like `.use` the most